client/store/auth.js

```diff
@@ -34,6 +33,7 @@ export const authenticate = (email, password, method) => async dispatch => {
     const res = await axios.post(`/auth/${method}`, {email, password})
     window.localStorage.setItem(TOKEN, res.data.token)
     dispatch(me())
+    history.push('/home');
```

Comment:
history.push here will redirect to home ONLY after login but not when you go to any other page